DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,17 @@ public function getConfigTreeBuilder()
             ->fixXmlConfig('extension')
             ->children()
                 ->arrayNode('security')
+                    ->children()
+                        ->arrayNode('roles')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->scalarNode('role_sonata_admin')
+                                    ->defaultValue('ROLE_SONATA_ADMIN')
+                                    ->info('Change the default role for sonata admin role')
```

Comment:
Wait I'm confusing with `ROLE_SUPER_ADMIN`… what is the purpose of this role exactly?